ts/src/time-profiler.ts

```diff
@@ -34,19 +34,27 @@ export interface TimeProfilerOptions {
   intervalMicros?: Microseconds;
   sourceMapper?: SourceMapper;
   name?: string;
+
+  /**
+   * This configuration option is experimental.
+   * When set to true, functions will be aggregated at the line-level, rather
```

Comment:
"line-level" -> "line level"?   because "function level" (and overall)

Reply:
Done (#73)